keanu-python/keanu/plates.py

```diff
@@ -0,0 +1,80 @@
+from typing import Callable, Generator, Dict, Optional, Any, Iterator
+
+from py4j.java_gateway import java_import
+
+from keanu.base import JavaObjectWrapper
+from keanu.context import KeanuContext
+from keanu.functional import Consumer
+from keanu.functional import BiConsumer
+from keanu.functional import JavaIterator
+from keanu.vertex import Vertex, cast_to_double_vertex, vertex_constructor_param_types
+from keanu.vertex.label import _VertexLabel
+
+k = KeanuContext()
+
+java_import(k.jvm_view(), "io.improbable.keanu.plating.Plates")
+java_import(k.jvm_view(), "io.improbable.keanu.plating.PlateBuilder")
+java_import(k.jvm_view(), "io.improbable.keanu.vertices.SimpleVertexDictionary")
+
+
+class Plate(JavaObjectWrapper):
+
+    def add(self, vertex: Vertex, label: Optional[str] = None) -> None:
+        if label is None:
+            self.unwrap().add(vertex.unwrap())
+        else:
+            self.unwrap().add(_VertexLabel(label).unwrap(), vertex.unwrap())
+
+    def get(self, label: str) -> Vertex:
+        return Vertex._from_java_vertex(self.unwrap().get(_VertexLabel(label).unwrap()))
+
+
+class Plates(JavaObjectWrapper):
```

Comment:
Can we add some python docs to clarify what the arguments do. Especially for: `factory` & `data_generator`

Reply:
done - let me know what you think.

Reply:
We should add `:raises ValueError`

Reply:
done.